backend/api/routes.py

```diff
@@ -58,49 +60,86 @@ def swagger_redirect():
     return RedirectResponse(url="/docs")
 
 
-@app.get("/version", tags=["info"])
+@app.get("/version", tags=["info"], description="Gets API version")
 def get_current_version():
     return app.version
 
 
 @app.get("/cdm", tags=["info"])
 def get_cdm():
+    """
```

Comment:
Could you move those to the api descriütion as well?

Reply:
Done.